docs/iris/example_code/Meteorology/wind_speed.py

```diff
@@ -0,0 +1,77 @@
+"""
+Plotting wind direction using quiver
+===========================================================
+
+This example demonstrates using quiver to plot wind speed
+
+The second plot uses normalised data to produce arrows with a uniform size
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import iris
+import iris.coord_categorisation
+import iris.quickplot as qplt
+
+import cartopy
+import cartopy.feature as cfeat
+import cartopy.crs as ccrs
+
+
+def main():
+    # Load the u and v components of wind from a pp file
+    infile = iris.sample_data_path('wind_speed_lake_victoria.pp')
+
+    uwind = iris.load_cube(infile, 'x_wind')
+    vwind = iris.load_cube(infile, 'y_wind')
+
+    ulon = uwind.coord('longitude')
+    vlon = vwind.coord('longitude')
+
+    # The longitude points go from 180 to 540, so subtract 360 from them
+    ulon.points = ulon.points - 360.0
+    vlon.points = vlon.points - 360.0
+
+    # Create a cube containing the wind speed
+    windspeed = (uwind ** 2 + vwind ** 2) ** 0.5
+    windspeed.rename('windspeed')
+
+    x = ulon.points
+    y = uwind.coord('latitude').points
+    u = uwind.data
+    v = vwind.data
+
+    # Set up axes to show the lake
+    lakes = cfeat.NaturalEarthFeature('physical', 'lakes', '50m',
+                                      facecolor='none')
+
+    ax = plt.axes(projection=ccrs.PlateCarree())
+    ax.add_feature(lakes)
+
+    # Plot the wind speed as a contour plot
+    qplt.contourf(windspeed, 20)
+
+    # Add arrows to show the wind vectors
+    plt.quiver(x, y, u, v, pivot='middle')
+
+    plt.title("Wind speed over Lake Victoria")
+    plt.show()
+
+    # Normalise the data for uniform arrow size
+    u_norm = u / np.sqrt(u ** 2.0 + v ** 2.0)
+    v_norm = v / np.sqrt(u ** 2.0 + v ** 2.0)
+
+    ax = plt.axes(projection=ccrs.PlateCarree())
+    ax.add_feature(lakes)
+
+    qplt.contourf(windspeed, 20)
+
+    plt.quiver(x, y, u_norm, v_norm, pivot='middle')
```

Comment:
I think this should probably have a `transform` keyword argument since the axes are a `GeoAxes` instance.